src/components/EventCalendar/Calendars.tsx

```diff
@@ -179,36 +105,37 @@ const Calendar: React.FC<CalendarProps> = ({
         >
           {date.getDate()}
           <div className={styles.list_box}>
-            {events
-              ?.filter((datas) => {
-                if (datas.startDate == dayjs(date).format('YYYY-MM-DD'))
-                  return datas;
-              })
-              .map((datas: Event) => {
-                return (
-                  <EventListCard
-                    key={datas._id}
-                    id={datas._id}
-                    eventLocation={datas.location}
-                    eventName={datas.title}
-                    eventDescription={datas.description}
-                    regDate={datas.startDate}
-                    regEndDate={datas.endDate}
-                    startTime={datas.startTime}
-                    endTime={datas.endTime}
-                    allDay={datas.allDay}
-                    recurring={datas.recurring}
-                    isPublic={datas.isPublic}
-                    isRegisterable={datas.isRegisterable}
-                  />
-                );
-              })}
+            {eventData &&
+              eventData
+                .filter((datas) => {
```

Comment:
The variable `datas` is not a good variable name here

Reply:
data is been used everywhere for now, i may open a new issue for this in the test and tsx files altogether. @Rumone